### PR TITLE
aishell: add new package

### PR DESCRIPTION
**Pull request by i-am-logger**
A simple Python code that connects to OpenAI's ChatGPT and executes the returned results.

### CODE REVIEW

File: pkgs/development/python-modules/aishell/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{ lib, python3Packages }:
```
Comment (dotlambda) [inline pkgs/development/python-modules/aishell/default.nix:1]:
```suggestion
{ lib, python3 }:
```

File: pkgs/development/python-modules/aishell/default.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+python3Packages.buildPythonPackage rec {
```
Comment (dotlambda) [inline pkgs/development/python-modules/aishell/default.nix:3]:
```suggestion
python3.pkgs.buildPythonApplication rec {
```

File: pkgs/development/python-modules/aishell/default.nix [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+  src = python3Packages.fetchPypi {
+    pname = "AiShell";
+    version = "0.1.0";
+    sha256 = "0abcdefghijklmnopqrstuvwxyz0123456789";
+    url = "https://github.com/code-yeongyu/AiShell/archive/${version}.tar.gz";
+  };
+
+  meta = {
+    description = "A simple Python code that connects to OpenAI's ChatGPT and executes the returned results.";
```
Comment (dotlambda) [inline pkgs/development/python-modules/aishell/default.nix:15]:
```suggestion
    description = "A simple Python code that connects to OpenAI's ChatGPT and executes the returned results";
```

File: pkgs/development/python-modules/aishell/default.nix [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "aishell";
+  version = "0.1.0";
```
Comment (dotlambda) [inline pkgs/development/python-modules/aishell/default.nix:5]:
```suggestion
  version = "2023.02.21";
```

**Review by dotlambda — CHANGES_REQUESTED** [review]
This is not a Python module and should thus be moved to somewhere under `pkgs/tools` and called from `pkgs/top-level/all-packages.nix`.

Please specify `format`, use `fetchFromGitHub`, and don't forget to add `propagatedBuildInputs`.

File: pkgs/tools/misc/aishell/default.nix [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  checkInputs = with python3.pkgs; [
+    pytest
+    pytest-cov
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A simple Python code that connects to OpenAI's ChatGPT and executes the returned results";
+    license = lib.licenses.mit;
```
Comment (azahi) [inline pkgs/tools/misc/aishell/default.nix:39]:
```suggestion
    license = licenses.mit;
```

**Comment by fabaff** [comment]
Depends on #217725

**Pull request closed without merging**